### PR TITLE
Admin UX/UI enhancement.

### PR DESCRIPTION
**Pull request by pavel-mironchik**

### CODE REVIEW

**Comment by pavel-mironchik** [comment]
[Delete item button - only if has no images (+warning)](https://trello.com/c/l4FFlB0j/50-delete-item-button-only-if-has-no-images-warning)

**Comment by pavel-mironchik** [comment]
[Preview button - new tab link](https://trello.com/c/5eQZTn1s/53-preview-button-new-tab-link)

**Comment by pavel-mironchik** [comment]
[Taxons UX/UI](https://trello.com/c/DM2UhyU8/54-taxons-ux-ui)

File: admin/src/pages/item.vue [hunk 1]
```diff
@@ -133,38 +165,40 @@
...
-                          v-on="on"
-                          icon
-                          @click="disableTaxonomyInheritance(taxon)"
-                        >
-                          <v-icon color="grey">mdi-lock-open</v-icon>
-                        </v-btn>
-                      </template>
-                      <span>Disable taxonomy inheritance</span>
-                    </v-tooltip>
+                    <template v-if="hasParent">
```
Comment (alquist42) [inline admin/src/pages/item.vue:168]: looks like hasParent is not defined

Comment (pavel-mironchik) [reply admin/src/pages/item.vue:168]: Thanks. Somehow lost it between branches.

**Pull request merged**